src/main/java/me/clip/placeholderapi/expansion/manager/LocalExpansionManager.java

```diff
@@ -320,6 +321,9 @@ private void registerAll(@NotNull final CommandSender sender) {
       Msg.msg(sender,
           registered == 0 ? "&6No expansions were registered!"
               : registered + "&a placeholder hooks successfully registered!");
+
+      final ExpansionsLoadedEvent event = new ExpansionsLoadedEvent();
```

Comment:
Why do it like that and not just `Bukkit.getPluginManager().callEvent(new ExpansionsLoadedEvent())`?

Reply:
You are right, I am gonna change it.

Reply:
Changed.